docs/design.md

```diff
@@ -145,21 +145,21 @@ service is fully configured by users.
 
 ### Jailing
 
+Seccomp filters are used by default to further limit the system calls Firecracker
+can use. There are 3 possible levels of seccomp filtering, configurable by passing
+a command line argument to Firecracker: 0 (disabled), 1 (whitelists a set of
+trusted system calls by their identifiers) and 2 (whitelists a set of trusted
+system calls with trusted parameter values), the latter being the most
+restrictive and the recommended one. The filters are loaded in the Firecracker
+process, immediately before the execution of the untrusted guest code starts.
```

Comment:
Since seccomp is completely decoupled from the jailer, could we move this to the `Seccomp` section ?

Reply:
Or split the jaling part into something like syscall jailing and memory and cpu jailing, since secomp is also a jailing

Reply:
Hmm, the jailer does more than memory and cpu jailing. it also does `chroot` and privilege dropping AFAIK. What do you say we rename the jailing section to sandboxing and create another level of hierarchy. I will explore this in the latest revision.

Reply:
done. @raduiliescu @serban300 WDYT?

Reply:
LGTM